.github/workflows/publish-img.yaml

```diff
@@ -48,6 +54,17 @@ jobs:
           tags: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:latest
           file: Dockerfile
 
+      - name: Push latest passt binding cni container image
+        if: github.repository_owner == 'kubevirt'
+        uses: docker/build-push-action@v5.3.0
+        with:
+          context: .
+          push: true
+          tags: ${{ env.REGISTRY }}/${{ env.PASST_BINDING_CNI_IMAGE_NAME }}:latest
+          file: passt/Dockerfile
+          build-args: |
+            KUBEVIRT_VERSION=${{ env.KUBEVIRT_VERSION }}
+
```

Comment:
Since we don't have nightlies this do not make much sense, also pushing "latest" explicitly is a bad practice since this is a docker implicit tag.

Reply:
we need it because it is the latest for the kubevirt ipam controller itself, as we have latest on the new manifest and on the kubevirt ipam controller (aka KIC) manifest itself

every push we publish both KIC and this image